doc/manual/developing_a_ndsp.rst

```diff
@@ -169,6 +169,26 @@ The program below exemplifies how to use logging: ::
         main()
 
 
+Integration with artiq experiments
+----------------------------------
+
+To integrate the controller into an experiment, simply add an entry into the ``device_db.py`` following the example shown below for the ``aqctl_hello.py`` from above: ::
+
+	device_db.update({
+    	"hello": {
+        	"type": "controller",
+        	"host": "::1",
+        	"port": 3249,
+        	"command": "python aqctl_hello.py -p {port}" 
+    	},
+	})
+	
+From the experiment this can now be added using ``self.setattr_device("hello")`` in the ``build()`` phase of the experiment, and methods accessed via: ::
+	
+	self.hello.message("Hello world!")
+
+For controllers hosted locally the ``artiq_ctlmgr`` utility from ``artiq-comtools`` can be used to automatically start the controllers listed in ``device_db.py``.
```

Comment:
locally?

Reply:
Surely artiq_ctlmgr can only start the controllers hosted on the machine the manager is being called from (whilst the device_db entry can provide access to controllers on other hosts)

Reply:
Yes, but you can run ctlmgr on multiple machines at the same time, and it should start the controllers that belong to each machine. Your sentence makes it sound like it would only support controllers that belong to the machine where the master is.